eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/SynchronizedBiMap.java

```diff
@@ -291,6 +292,20 @@ public V forcePut(K key, V value)
         }
     }
 
+    @Override
+    public MutableBiMap<K, V> withMap(Map<K, V> map)
+    {
+        if (map == null)
+        {
+            throw new IllegalArgumentException("Cannot Add a null Map!");
+        }
+        synchronized (this.lock)
+        {
+            map.forEach(this::put);
+            return this;
+        }
```

Comment:
Synchronized wrappers should delegate to methods of the same name. But then return `this`, the wrapper.